pkg/features/kube_features.go

```diff
@@ -601,6 +601,12 @@ const (
 	// Allow setting the Fully Qualified Domain Name (FQDN) in the hostname of a Pod. If a Pod does not
 	// have FQDN, this feature has no effect.
 	SetHostnameAsFQDN featuregate.Feature = "SetHostnameAsFQDN"
+
+	// owner: @RenaudWasTaken @dashpole
+	// alpha: v1.19
```

Comment:
I wonder what will make this feature go to beta? It looks quite straightforward. Is it alpha just from process standpoint?

Reply:
sorry, found it myself: https://github.com/RenaudWasTaken/enhancements/blob/6b4b9d00f22d2e3524be3b184e17f9340e0b3457/keps/sig-node/1867-disable-accelerator-usage-metrics/README.md#alpha---beta-graduation

Reply:
Yep we discussed this within sig-node a bit, we even thought about enabling it by default. It's definitely alpha from a process standpoint but also to start warning users that this feature is happening.

It also gives users a bit of time to start opening issues if the impact is much more important than we anticipated before it's enabled by default!